pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -76,6 +77,9 @@ def __init__(self, runtime: Runtime, group: str, assembly: str, data_path: str,
         self._doozer_env_vars["DOOZER_DATA_PATH"] = data_path if data_path else \
             self.runtime.config.get("build_config", {}).get("ocp_build_data_url")
 
+        if self.skip_get_nightlies and len(self.nightlies) != 4:
+            raise ValueError("When using --skip-get-nightlies, nightlies for all 4 arches must be specified")
```

Comment:
```suggestion
        if self.skip_get_nightlies and len(self.nightlies) != len(self.arches):
            raise ValueError(f"When using --skip-get-nightlies, nightlies for all given {len(self.arches)} arches must be specified")
```
we allow arches to be less than 4 so this would be accurate